tests/basic/overlay/Makefile

```diff
@@ -0,0 +1,40 @@
+include ../../common.mk
+
+PROG=/bin/touch /foo.txt
```

Comment:
It would be good to check that the overlayfs semantics is actually working.

Reply:
Well, the semantics is that now it's writable. If it's read-only then touch will fail which I checked.